### PR TITLE
Fix schema.ts

### PR DESCRIPTION
**Pull request by kimhxsong**
Please consider the following request.

If the administrator account in PostgreSQL is not named 'postgres' and the password is not 'root', an error occurs when calling the execute(...) function.

This case was taken into account when calling execute in the main function (line 34), but it was not handled in the third call (line 54). Therefore, I have corrected the code as in the first call.

### Problem Situation
- docker-compose.yml
```yaml
version: "3"

services:
  postgres:
    image: postgres:latest
    container_name: postgres
    restart: always
    ports:
      - ${postgres_container_port}:5432
    environment:
      postgres_database: ${postgres_database}
      postgres_user: ${postgres_user}
      postgres_password: ${postgres_password}
      TZ: Asia/Seoul
    volumes:
      - data:/var/lib/postgresql/data
    init: true
    tty: true

  pgadmin:
    container_name: pgadmin
    image: dpage/pgadmin4
    restart: always
    volumes:
      - admin:/var/lib/pgadmin
    ports:
      - ${PGADMIN_HOST_PORT}:${PGADMIN_CONTAINER_PORT}
    environment:
      pgadmin_default_email: ${pgadmin_default_email}
      pgadmin_default_password: ${pgadmin_default_password}
      TZ: Asia/Seoul
    init: true

volumes:
  data: {}
```
- .env
```
POSTGRES_HOST=postgres
POSTGRES_DATABASE=postgres
POSTGRES_USER=test
postgres_password=4242
POSTGRES_HOST_PORT=4242
postgres_container_port=5432

PGADMIN_HOST_PORT=8080
pgadmin_container_port=80
PGADMIN_DEFAULT_EMAIL=postgres@mail.com
pgadmin_default_password=4242
```

I have configured the docker-compose file as above and made modifications to DB_ACCOUNT, DB_NAME, etc. as described in the README.md file. Then, I executed the following commands in sequence:

```bash
$> npm i
$> npm build
$> npm schema test 4242
```

After execution, I encountered the following error:
```
error: password authentication failed for user "postgres"
    at Parser.parseErrorMessage (/Users/hyeonsok/Dev/myback/backend/node_modules/pg-protocol/src/parser.ts:369:69)
    at Parser.handlePacket (/Users/hyeonsok/Dev/myback/backend/node_modules/pg-protocol/src/parser.ts:188:21)
    at Parser.parse (/Users/hyeonsok/Dev/myback/backend/node_modules/pg-protocol/src/parser.ts:103:30)
    at Socket.<anonymous> (/Users/hyeonsok/Dev/myback/backend/node_modules/pg-protocol/src/index.ts:7:48)
    at Socket.emit (node:events:511:28)
    at addChunk (node:internal/streams/readable:332:12)
    at readableAddChunk (node:internal/streams/readable:305:9)
    at Socket.Readable.push (node:internal/streams/readable:242:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 104,
  severity: 'FATAL',
  code: '28P01',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'auth.c',
  line

: '326',
  routine: 'auth_failed'
}
```

The issue seems to be caused by passing 'postgres' as the username in the above code snippet in schema.ts (lines 13-18).
Could you please review it?

Thank you.

### CODE REVIEW

**Review by samchon — CHANGES_REQUESTED** [review]
Good edit, but would be much better if use `??` symbol.

**Comment by kimhxsong** [comment]
Thanks for pointing that out. As you said, using the "??" symbol is definitely a better option.

---

### There is one more thing I would like to correct.

It looks like the admin database name needs to be 'postgres' in PostgreSQL for the current code to work without modification.

```bash
npm execute schema [ROOT_USERNAME ?? postgres ] [USER_SECRET ?? root]
```

I'd like to ask your opinion on adding an argument for the name to that command, modifying it to something like this

```bash
$> npm run schema [ROOT_USERNAME ?? 'postgres'] [ROOT_SECRET ?? 'root' ] [DEFAULT_DB_NAME ?? 'postgres' ] [DB_HOST ?? '127.0.0.1']
### OR
$> npm run setup [ROOT_USERNAME ?? 'postgres'] [ROOT_SECRET ?? 'root' ] [DEFAULT_DB_NAME ?? 'postgres' ] [DB_HOST ?? '127.0.0.1']
```

The `README.md` file (line 150) mentions the `npm run setup` command, but it doesn't exist in the package.json. I checked the `.github/workflow/build.yml` and found that the schema command initializes the database. 

Considering this, I think you might want to consider renaming the schema command to 'setup' or adding a 'setup' command that can take more arguments.

To summarize
1. no modifications.
2. modify the schema command to accept additional arguments.
3. do 2. and rename the schema command to setup.
4. add the setup command and make the schema command accept additional arguments.

If your comment falls into one of the 2-4 categories and you don't mind modifying it, I'll commit it and ask you to review it again.

**Comment by samchon** [comment]
I think that it would better to reconstruct arguments like below:

```bash
npm run schema -- --host HOST --username USERNAME --password --PASSWORD
```

If you send a PR about that, I'll accept. If not, I'll remake it until Wednesday.

At Wednesday, I'm planning to use `commander` module like below code:

https://github.com/samchon/nestia/blob/master/packages/cli/src/internal/ArgumentParser.ts#L15-L19

**Comment by kimhxsong** [comment]
`README.md` (line 110) The description of the `npm schema` command looks like it needs to be fixed, so you might as well do it yourself.

For `commander.options(...)`, I didn't know whether `[...]` or `<...>` was appropriate, so I arbitrarily set it. You can check it out and modify it to what you originally intended.

**Comment by samchon** [comment]
Sorry for delaying, but may changed as you wish especially about `.env` file.
